source/frontend/src/features/properties/map/compensation/update/UpdateCompensationRequisitionForm.tsx

```diff
@@ -0,0 +1,182 @@
+  isLoading,
+  formikRef,
+  acquisitionFile,
+  initialValues,
+  onSave,
+  onCancel,
+}) => {
+  const { setModalContent, setDisplayModal } = useModalContext();
+  let fiscalYearOptions = generateFiscalYearOptions();
+
+  const cancelFunc = (resetForm: () => void, dirty: boolean) => {
+    if (!dirty) {
+      resetForm();
+      onCancel();
+    } else {
+      setModalContent({
+        ...getCancelModalProps(),
+        handleOk: () => {
+          resetForm();
+          setDisplayModal(false);
+          onCancel();
+        },
+      });
+      setDisplayModal(true);
+    }
+  };
+
+  return (
+    <StyledFormWrapper>
+      <Formik<CompensationRequisitionFormModel>
+        enableReinitialize
+        innerRef={formikRef}
+        initialValues={initialValues}
+        onSubmit={async values => {
+          await onSave(values);
+        }}
+        validationSchema={CompensationRequisitionYupSchema}
+      >
+        {formikProps => (
+          <>
+            <LoadingBackdrop show={isLoading}></LoadingBackdrop>
+            <UnsavedChangesPrompt />
+
+            <Section header="Requisition details">
+              <SectionField label="Status" labelWidth="4" contentWidth="4">
+                <Select
+                  field="status"
+                  options={[
+                    {
+                      label: 'Draft',
+                      value: 'draft',
+                    },
+                    {
+                      label: 'Final',
+                      value: 'final',
+                    },
+                  ]}
+                  placeholder="Select..."
+                />
+              </SectionField>
+              <SectionField label="Agreement date" labelWidth="4" contentWidth="4">
+                <FastDatePicker field="agreementDateTime" formikProps={formikProps} />
+              </SectionField>
+              <SectionField
+                label="Expropriation notice served date"
+                labelWidth="4"
+                contentWidth="4"
+              >
+                <FastDatePicker
+                  field="expropriationNoticeServedDateTime"
+                  formikProps={formikProps}
+                />
+              </SectionField>
+              <SectionField label="Expropriation vesting date" labelWidth="4" contentWidth="4">
+                <FastDatePicker field="expropriationVestingDateTime" formikProps={formikProps} />
+              </SectionField>
+              <SectionField label="Special instructions" labelWidth="12">
+                <MediumTextArea field="specialInstruction" />
+              </SectionField>
+            </Section>
+            <Section header="Financial coding">
+              <SectionField label="Product" labelWidth={'4'}>
+                {acquisitionFile.product?.code ?? ''}
+              </SectionField>
+              <SectionField label="Business function" labelWidth={'4'}>
+                {acquisitionFile.project?.code ?? ''}
```

Comment:
shouldn't this be the project.businessFunctionCode ?

Reply:
updated.